packages/create-svelte/bin.js

```diff
@@ -196,12 +215,23 @@ function write_common_files(cwd, options, name) {
 			const new_pkg = JSON.parse(file.contents);
 			merge(pkg, new_pkg);
 		} else {
+			if (file.name === 'svelte.config.js') {
+				file.contents = file.contents
+					.replace(
+						'// insert:adapter-import\n',
+						options.adapter ? `import adapter from '${options.adapter.npm}';\n` : ''
+					)
+					.replace(/\t+\/\/ insert:adapter/, options.adapter ? '\t\tadapter: adapter(),' : '');
+			}
 			const dest = path.join(cwd, file.name);
 			mkdirp(path.dirname(dest));
 			fs.writeFileSync(dest, file.contents);
 		}
 	});
 
+	if (options.adapter) {
+		pkg.devDependencies[options.adapter.npm] = 'next';
```

Comment:
Should the version be the actual value of the latest release instead of the tag? 
People installing like this would get left behind once adapters are released as 1.0 which switches to latest tag.

As a workaround for having to pull every adapter version for now we could use "^1.0.0-next.0"  which should match all releases of the adapters including "1.x.y". It won't match new majors after 1 eg "2.x.y" or new prereleases after 1.0.0 eg "1.1.2-next.1"

Reply:
@dummdidumm I tried using the shared files for this but would end up making 10 different svelte.config.js and that would make future changes more difficult. Can use it for adding adapter to devDependencies but the current logic will still be needed if we want to include community adapters as discussed here https://github.com/sveltejs/kit/issues/2459#issuecomment-925236207